code/modules/admin/topic.dm

```diff
@@ -802,34 +802,59 @@ Status: [status ? status : "Unknown"] | Damage: [health ? health : "None"]
 			return
 
 		var/subject = input("Enter the subject line", "Fax Message", "") as text|null
-
-		var/fax_message
-		var/addressed_to
-		var/type = input("Do you want to use the template or type a custom message?", "Template") as null|anything in list("Template", "Custom")
+		var/type = input("Do you want to use the template, pencode or type a raw message?", "Fax Message") as null|anything in list("Template", "Pencode", "Raw")
```

Comment:
There are two fax functions, don't forget to modify the other one too (one is respond, one is create fax)

Reply:
I ended up combining both to remove some code duplication